src/components/JsonSchemaViewer.tsx

```diff
@@ -113,19 +98,16 @@ export class JsonSchemaViewerComponent extends React.PureComponent<IJsonSchemaVi
 
   public render() {
     const {
-      props: { emptyText = 'No schema defined', name, schema, expanded, defaultExpandedDepth, className, ...props },
+      props: { schema, expanded, defaultExpandedDepth, className, ...props },
     } = this;
 
-    // an empty array or object is still a valid response, schema is ONLY really empty when a combiner type has no information
```

Comment:
TODO: bring this back? Is this actually useful in any meaningful way?

Reply:
@marbemac was that "empty schema screen" valuable?

Reply:
I think it's helpful, otherwise it just renders nothing right? If we don't do here, we'll have to handle in places that use JSV so that it's not just blank to end user. Maybe just a prop that accepts a `React.ReactNode` to render when schema is otherwise empty? including if `oneOf: []`, etc (e.g. would end up rendering nothing). What do you think?

Reply:
Sure, will add it back.